cache/search_stats.go

```diff
@@ -501,6 +501,32 @@ func (ssc *SearchStatsCacheImpl) loadSourcesByPositionAndParent() (map[string]st
 		key := fmt.Sprintf("%v-%v-%v", parent_uid, position, type_id)
 		ret[key] = source_uid
 	}
+
+	query = `select p.uid as parent_uid, gc.uid as source_uid, gc.type_id 
+	from sources p
+		join sources c on c.parent_id = p.id
+		join sources gc on gc.parent_id = c.id
+	where c.position is not null and p.name = 'TES'
```

Comment:
Do not check the source by name (p.name = 'TES'). Names may be changed in the future.
You should check sources by UID.

Reply:
fixed